ivy/functional/backends/torch/general.py

```diff
@@ -324,7 +324,7 @@ def inplace_update(
         if is_variable(x_native):
             x_native.copy_ = val_native
         else:
-            x_native[()] = val_native
+            x_native = val_native
```

Comment:
let's probably revert this change, might be a problem with the code that uses `inplace_update`

Reply:
Done 👍🏻

Reply:
awesome, thanks!